src/app/rpmostree-builtin-status.cxx

```diff
@@ -302,6 +379,17 @@ print_daemon_state (RPMOSTreeSysroot *sysroot_proxy,
                  update_driver_name, update_driver_sd_unit);
       else
         g_print ("AutomaticUpdates: driven by %s\n", update_driver_name);
+
+      /* only try to get unit's StatusText if we're on the system bus */
+      if (bus_type == G_BUS_TYPE_SYSTEM)
+        {
+          g_autofree const char *update_driver_state = NULL;
+          if (!get_update_driver_state (sysroot_proxy, update_driver_sd_unit,
+                                        &update_driver_state, cancellable, error))
+            g_warning ("%s", (*error)->message);
```

Comment:
We need to "take ownership" of the error in this case, and not pass it back up to the caller.  Something more like:
```
g_autoptr(GError) local_error = NULL;
if (!get_update_driver_state (sysroot_proxy, update_driver_sd_unit,
                              &update_driver_state, cancellable, &local_error))
  g_warning ("%s", local_error->message);
```

Reply:
I think we should also use `g_printerr ("Failed to get update driver state: %s\n", local_error->message)` over `g_warning()` which...may actually trigger our usage of `export G_DEBUG=fatal-warnings` in some cases.

Reply:
ohh I see. Learned a lot about errors in GLib in this PR :)